### PR TITLE
export scripts for imdb_wiki data

### PR DESCRIPTION
**Pull request by johndon98**
@atimin

### CODE REVIEW

File: export/imdb/export.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        dataset_name = foldername
+        counter = 0
+        folder_path = os.path.join(path, dataset_name)
+        for image in os.listdir(folder_path):
+            img_path = os.path.join(folder_path, image)
+            with open(img_path, 'rb') as f:
+                img_data = f.read()
+                for i, img_in_meta_data in enumerate([img['full_path'][3:] for img in meta_data_list]):
+                    if image == img_in_meta_data:
+                        label = meta_data_list[i]
```
Comment (atimin) [inline export/imdb/export.py]: Didn't manage to make it work. `if image == img_in_meta_data` is always false. I need the images with metadata so if there is no one, then skip image. Moreover, it is very slow. You use a list of dictionaries and search in the loop again and again. It takes 1 second on my laptop. So, the whole export could take 5 days... Try to change your `mat_to_dict` function and return a dictionary where the key is a filename and the value is metadata.

File: export/imdb/export.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+                    else:
+                        label = None
+
+                labels = {
+                    'label': label
```
Comment (atimin) [inline export/imdb/export.py]: if the `lable` variables is already a dictionary use it directly:

```
await bucket.write(REDUCT_STORE_ENTRY + dataset_name, img_data, timestamp=counter, labels=label,
                                   content_type="image/jpg")
```

**Review by atimin — CHANGES_REQUESTED** [review]

File: utils/meta_data_parser.py [hunk 3]
```diff
@@ -1,35 +1,11 @@
...
-    meta_data_each_image = []
-
-    for i, x in imdb_meta_data.iterrows():
-        meta_data_each_image.append(x.to_dict())
-
-    return meta_data_each_image
\ No newline at end of file
+    # there is already a 'name' field included and 'celeb_names' is a duplicate.
+    file.pop('celeb_names')
+    for row in pd.DataFrame(file).to_dict('records'):
```
Comment (johndon98) [inline utils/meta_data_parser.py:9]: Here, I convert a dict with arrays as values into a dataframe and then obtain a dict of dicts (where each dict is a row of the dataframe). This way meta data for each image is in the inner_dicts within one larger dict.

File: utils/meta_data_parser.py [hunk 4]
```diff
@@ -1,35 +1,11 @@
...
+        meta_data_dict[row['full_path'][3:]] = row
```
Comment (johndon98) [inline utils/meta_data_parser.py:10]: Here I use the filename of the image as key and the values are inner_dicts containing meta data of each image.

File: utils/meta_data_parser.py (hunk 3 above)
Comment (johndon98) [reply utils/meta_data_parser.py:9]: Is there a faster way to obtain this structure {filename: {meta_data}} ?
Please let me know if you have suggestions to obtain the desired final format using a different method.
I am not sure if converting a dict (with arrays as values) to dataframe and then back to individual dict is the most efficient way. @atimin

File: utils/meta_data_parser.py (hunk 4 above)
Comment (atimin) [reply utils/meta_data_parser.py:10]: It should be more efficient than manual search in a loop. You can also see it if you run it locally. If you didn't manage to setup an instance locally, I can provide a bucket for testing on https://play.reduct.store

**Review by atimin — APPROVED** [review]

File: utils/meta_data_parser.py (hunk 4 above)
Comment (johndon98) [reply utils/meta_data_parser.py:10]: Yes, I already did a test run. I setup reductstore locally using the docker image. Then I created a bucket and wrote the image files into this bucket. This is how I got to know it took approx. 2 hours for 30GB. 
Let me know if the export is faster for you now.

Comment (atimin) [reply utils/meta_data_parser.py:10]: 2 hours is ok, I think. I'll optimize it later if it is necessary. Let me try it on my side , and we're done. Thank you!

Comment (johndon98) [reply utils/meta_data_parser.py:10]: Yes, sounds good. Thank you.

**Pull request merged**